<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />

  <node pkg="tf2_ros" type="static_transform_publisher" name="virtual_joint_broadcaster_1" args="0 0 0 0 0 0 world panda_link0" />

  <include file="$(find franka_control)/launch/franka_control.launch">
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="false" />
  </include>
  <include file="$(find panda_moveit_config)/launch/panda_moveit.launch">
    <arg name="load_gripper" value="false" />
  </include>

  <node name="move_to_start" pkg="franka_example_controllers" type="move_to_start.py" output="screen" required="true" />

</launch>
